internal/action/bindings.go

```diff
@@ -251,6 +251,27 @@ func findEvent(k string) (Event, error) {
 	return event, nil
 }
 
+func compareEvents(a Event, b Event) bool {
```

Comment:
nit: Maybe something like `eventsEqual()` (to make it unambiguous what is the return value)?